torch/_dynamo/output_graph.py

```diff
@@ -541,16 +529,42 @@ def call_user_compiler(self, gm: fx.GraphModule) -> CompiledFn:
             )
             _step_logger()(logging.INFO, f"calling compiler function {name}")
             compiler_fn = self.compiler_fn
+            # WrapperBackend needs real inputs, for now, to verify correctness
             if config.verify_correctness:
-                compiler_fn = wrap_compiler_fn(compiler_fn)
-            compiled_fn = compiler_fn(gm, self.example_inputs())
+                compiler_fn = WrapperBackend(compiler_fn, self.example_inputs())
+
+            # We invoke the backend with real inputs if and only if
+            # we are in doing accuracy evaluation in the minifier
+            # Note: This check is the same as in `save_graph_repro`
+            is_minifier_backend = "_accuracy" in str(
```

Comment:
nuh uh, we're not testing for minifier backend this way. Why can't you just do an `is` test here, there's what, one function you need to test?